docs/content/preview/admin/yb-admin.md

```diff
@@ -461,6 +461,36 @@ Waiting for compaction...
 Compaction complete: SUCCESS
 ```
 
+#### compact_table_by_id
+
+Triggers manual compaction on a table.
+
+**Syntax**
+
+```sh
+yb-admin \
+    -master_addresses <master-addresses> \
+    compact_table_by_id <table_id> \
+    [timeout_in_seconds] [ADD_INDEXES]
+```
+
+* *master-addresses*: Comma-separated list of YB-Master hosts and ports. Default value is `localhost:7100`.
+* *table_id*: The unique UUID associated with the table to be compacted.
+* *timeout_in_seconds*: Specifies duration, in seconds, yb-admin waits for compaction to end. Default value is `20`.
```

Comment:
timeout change should also be applied to compact_table above

Reply:
What do you mean here? `compact_table` already has it?

Reply:
I just mean the wording change. you originally copied the wording from compact_table, copy my edits back to compact_table. "cli timeouts" isn't so great grammatically speaking